### ISSUE TITLE
Upload: A11y issues

### ISSUE THREAD

**Issue by JoseAntonioNavasSolutia** [primefaces/primevue#5268]
### Describe the bug

NVDA screen reader does not read "input type="file"

### Reproducer

.

### PrimeVue version

3.48.1

### Vue version

3.x

### Language

ES6

### Build / Runtime

Nuxt
